src/Components/SearchResults/SearchResults.css

```diff
@@ -66,22 +67,34 @@ h3 {
   .search-results-content {
     flex-direction: column;
   }
+
+  .specific-search-result {
+    width: 95%;
+  }
 }
-@media only screen and (min-width: 601px) {
+@media only screen and (min-width: 601px) and (max-width: 767px) {
```

Comment:
This might be a code smell, but I noticed that even at screen sizes where I would expect these rules not to apply, they were still being applied. For example at 840px instead of using the below rules for 768px and above it was using these rules. I'll have to look into this in the future to see if there was a different issue causing this problem and a better solution.